stanford_alt_check.forms.inc

```diff
@@ -2,124 +2,532 @@
 
 /**
  * @file
- * Stanford Alt Check Forms.
+ * Stanford Alt Check Upload Form.
  */
 
 /**
- * Alt Check Images form callback.
+ * Stanford Alt check edit form.
+ */
+function stanford_alt_check_form($form, &$form_state) {
+  if (module_exists('colorbox')) {
```

Comment:
I'm using `colorbox-load` class on the <a> tag on the image. The thumbnail image links to the full size image in case the user can't tell what the contents are. The colorbox could help a user to view a full size image without opening a new window when they click on the image.